Sources/SourceKitLSP/Swift/SwiftTestingScanner.swift

```diff
@@ -292,7 +292,7 @@ final class SyntacticSwiftTestingTestScanner: SyntaxVisitor {
     }
 
     let name =
-      node.name.text + "(" + node.signature.parameterClause.parameters.map { "\($0.firstName):" }.joined() + ")"
```

Comment:
So, this is still not quite correct because of optional leading/trailing backticks. See https://github.com/apple/swift-syntax/issues/1936.

swift-testing has logic to trim these off that you may need to adopt.

Reply:
So looks like the identifier text contains the backticks in the TokenSyntax, which makes sense since \`foo\` is distinct from foo. @ahoppen is there a way to grab the name without resorting to string manipulation? @grynspan pointed me to https://github.com/apple/swift-testing/blob/db5a6461c12be87eed59a87a6e557193252c83fb/Sources/TestingMacros/Support/Additions/TokenSyntaxAdditions.swift#L22

Reply:
Seems like the best course of action is to leverage https://github.com/apple/swift-syntax/pull/2576 when it is ready. @ahoppen  In the short term is it worth stripping the backticks in the TestItem initializer until the proper solution is merged?

Reply:
@ahoppen @grynspan I think its worth merging this on its own and addressing the backticks in a follow up.

Reply:
Created #1211 as a stopgap until https://github.com/apple/swift-syntax/pull/2576 is ready